Gandalf/build.sh

```diff
@@ -1,119 +1,98 @@
-packagedescription='Some tweaks may break jailbreaks. Let this tweak say "You Shall Not Pass!" to incompatible tweaks and you can sit back and have fun with your jailbreak.'
-packagedepiction='https://ethanrdoesmc.github.io/gandalf/docs/depictions/?p=io.github.ethanrdoesmc.gandalf102'
-maintainer="EthanRDoesMC <ethanrdoesmc@gmail.com>"
-author="EthanRDoesMC <ethanrdoesmc@gmail.com>"
-depends="firmware (>=10.0), sudo, com.officialscheduler.mterminal"
-section="Manager_Addons"
+#Script specific variables.
+GANDALF_COMMAND_NAME="gandalf102"
 
-# Show user what he's trying to compile
+#Main script
 
+#Start message
+echo "Started packaging ${PKG_NAME}"
 
-echo "Welcome to $productname! You are now compiling $basename; Version $packageversion. The name '$packagename' will be shown in Cydia. The name of the hold script will be '$scriptname'. I hope that's all correct. If not: please abort this script within 3 seconds. Otherwise just wait."
+#Prepare the package structure
+echo "Creating package structure..."
 
-sleep 3
+mkdir "${PKG_PACKAGE}"
+mkdir "${PKG_PACKAGE}/DEBIAN"
+mkdir -p "${PKG_PACKAGE}/usr/bin"
+mkdir -p "${PKG_PACKAGE}/var/mobile/Downloads/Gandalf102"
 
-# create control file and setup for conflicts
-echo "Creating directories and copying files..."
-# Create control file
-mkdir -p "$basename/DEBIAN/"
-controlfile="$basename/DEBIAN/control"
-touch $controlfile #create it (just for sure)
-echo "Package: $basename" > $controlfile #Delete whole content of the file and add identifier
-echo "Name: $packagename" >> $controlfile
-echo "Version: $packageversion" >> $controlfile
-echo "Architecture: iphoneos-arm" >> $controlfile #better not to change this...
-echo "Replaces: $packagereplaces" >> $controlfile
-echo "Description: $packagedescription" >> $controlfile
-echo "Depiction: $packagedepiction" >> $controlfile
-echo "Maintainer: $maintainer" >> $controlfile
-echo "Author: $author" >> $controlfile
-echo "Depends: $depends" >> $controlfile
-echo "Section: $section" >> $controlfile
 
-#for debugging:
-# cat $controlfile
+#Create the control file
+echo "Creating the control file..."
 
-# cp control102 $controlfile
+cat <<EOF > "${PKG_PACKAGE}/DEBIAN/control";
+Package: ${PKG_PACKAGE}
+Name: ${PKG_NAME}
+Version: ${PKG_VERSION}
+Architecture: ${PKG_ARCHITECTURE}
+Replaces: ${PKG_REPLACES}
+Description: ${PKG_DESCRIPTION}
+Depiction: ${PKG_DEPICTION}
+Maintainer: ${PKG_MAINTAINER}
+Author: ${PKG_AUTHOR}
+Depends: ${PKG_DEPENDS}
+Section: ${PKG_SECTION}
+EOF
 
-#CODE CLEANUP NEEDS TO HAPPEN. -> Define all variables at beginning?
-#We aren't adhering to standards very well.
 
+#Compress the application
+echo "Compressing and moving Gandalf.app..."
 
-# parse conflicts
-# works on RHEL/CentOS -1Conan
-conflicts=$(cat $conflictfile | sed ':a;N;$!ba;s/\n/,\ /g')
-echo "Breaks: ${conflicts}" >> $controlfile
-#for debugging:
+tar -czf "Gandalf.app.tar.gz" "gandalf.app"
+mv "Gandalf.app.tar.gz" "${PKG_PACKAGE}/var/mobile/Downloads/Gandalf102"
```

Comment:
Yep. And on /Gandalf there are 2 .apps 

1. Gandalf.app
2. gandalf.app

What's the difference?